src/app/datos.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Pokemon } from './pokemonsInterface'; /* escrito a mano */
+
+
+@Injectable({
+  providedIn: 'root',
+})
+
+export class DatosService {
+  
+  constructor(private http: HttpClient) { }
+  URL_LISTA_POKEMON: string = 'https://pokeapi.co/api/v2/pokemon?limit=151&offset=0';
+  listaPokemonsVisibles: Pokemon[] = [];
+  listaPokemonsCuestionario: Pokemon[] = [];
+
+
+
+  ejecutar() {
+
+    for (var i = 0; i <= 3; ++i) {
+      if (this.listaPokemonsVisibles[i].nombre = undefined){
+        this.listaPokemonsVisibles[i].nombre = "holi"
+      }
+      var numeroAleatorio = Math.floor(Math.random() * (150)); /* en este caso son 150 Pokemons los que queremos indexar. Pero se pueden muchísimos más */
```

Comment:
Esto mismo, generar un número aleatorio en cada pasada del `for`, fue lo que hicimos al principio de la otra pull request, pero fallaba (y falla aquí tb) porque para cuando llega la respuesta del primer GET que hicimos, la variable `numeroAleatorio` ya ha cambiado de valor varias veces

Lo corregiste en el commit https://github.com/CarlosCaNav/PokemonAPI/pull/1/commits/18eba8581538e5bc3f515e34c4a45a44be24af33 generando un array de números aleatorios de antemano, y aquí tenemos que hacer lo mismo